ladybug_geometry/intersection2d.py

```diff
@@ -57,12 +57,10 @@ def intersect_line2d(line_ray_a, line_ray_b):
 
 def intersect_line2d_infinite(line_ray_a, line_ray_b):
     """Get the intersection between a Ray2D/LineSegment2D and another extended infinitely.
-
```

Comment:
If you delete this line and the other in this file, I think it may not be formatted correctly by Sphinx for the online docs.